articles/iot-edge/how-to-deploy-cli-at-scale.md

```diff
@@ -162,6 +162,23 @@ The previous example showed a layered deployment setting the `properties.desired
 }
 ```
 
+The same can also be expressed with:
+
+```json
+"SimulatedTEmperatureSensor": {
+  "properties.desired.layeredProperties.SendData" : true,
+  "properties.desired.layeredProperties.SendInterval": 5
+}
+```
+
+> Warning: Currently there is a bug with a layered deployment that updates only properties. These layers need an empty edgeAgent object (i.e. `"$edgeAgent":{}`) in the deployment manifest to be considered a valid layered deployment. They will also only be shown as applied in the module twins they target. For the `$edgeAgent` module twin, and hence in the Azure portal, the layer will only ever be marked as targeted, not applied.
+
+In summary, to create a layered deployment:
+
+- Must add the `--layered` flag to the Azure CLI create command
+- It may not contain system modules
+- Must use the 'dot notation' under `$edgeAgent` and `$edgeHub`
```

Comment:
Can you explain what you mean by "dot notation"?

Reply:
I've added a clarifying example now. I'm also totally open to a better word to explain the pattern 😄

Reply:
I believe that the two examples you included are just different ways of writing the same thing. You don't have to collapse the sections, it's just a shorthand. 

```
"$edgeAgent.properties.desired.modules.SimulatedTemperatureSensor": {
```

Should resolve to be the same as

```json
"$edgeAgent": {
  "properties": {
    "desired": {
      "modules": {
        "SimulatedTemperatureSensor": {
```

Have you found that's not the case?

Reply:
@kgremban I've added your suggestion as discussed